src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs

```diff
@@ -35,14 +35,32 @@ public DisputeController(IOracleDataApi_v1_0Client oracleDataApi, ILogger<Disput
     /// <param name="cancellationToken"></param>
     /// <returns></returns>
     [HttpGet("/dispute/{disputeId}")]
-    [ProducesResponseType(typeof(Dispute), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(Dispute), (int)HttpStatusCode.OK)]
+    [ProducesResponseType((int)HttpStatusCode.NotFound)]
+    [ProducesResponseType((int)HttpStatusCode.InternalServerError)]
     public async Task<IActionResult> GetDisputeAsync(int disputeId, CancellationToken cancellationToken)
     {
-        _logger.LogDebug("Retrieving Dispute from oraface-api");
+        _logger.LogDebug("Retrieving Dispute from oracle-data-api");
 
-        Dispute dispute = await _oracleDataApi.GetDisputeAsync(disputeId, cancellationToken);
-
-        return Ok(dispute);
+        try
+        {
+            Dispute dispute = await _oracleDataApi.GetDisputeAsync(disputeId, cancellationToken);
+            return Ok(dispute);
+        }
+        catch (TrafficCourts.Staff.Service.OpenAPIs.OracleDataApi.v1_0.ApiException e) when (e.StatusCode == (int) HttpStatusCode.NotFound)
+        {
+            return NotFound();
+        }
+        catch (TrafficCourts.Staff.Service.OpenAPIs.OracleDataApi.v1_0.ApiException e)

```

Comment:
Should we catch 400 Bad Request exception as well?

Reply:
Possibly.  But I haven't been able to reproduce a 400, not sure it's ever thrown.

Reply:
If I send request by not providing the int dispute id and provide characters instead on the path, I get 400 bad request error. For example: http://localhost:5010/api/v1.0/dispute/asdf

Reply:
Yup, just did the same.  This looks to be inherently caught because the type of (int) doesn't match the path parameter {disputeId}

`https://localhost:7136/dispute/asdf`
```
{
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
    "title": "One or more validation errors occurred.",
    "status": 400,
    "traceId": "00-9fe15328b23528b08c588ce7be1fddda-20605d1b29f5552d-00",
    "errors": {
        "disputeId": [
            "The value 'asdf' is not valid."
        ]
    }
}
```

Reply:
But I'll add it anyway to be clear what we're catching.